### ISSUE TITLE
Using Metric depth

### ISSUE THREAD

**Issue by ducha-aiki** [LiheYoung/Depth-Anything#9]
Hi,

Can I use metric depth checkpoints with provided code snippet

https://github.com/LiheYoung/Depth-Anything/tree/main#import-depth-anything-to-your-project

or should I only use it through the ZoeDepth methods?

```python3
from depth_anything.dpt import DPT_DINOv2
from depth_anything.util.transform import Resize, NormalizeImage, PrepareForNet

import cv2
import torch

depth_anything = DPT_DINOv2(encoder='vitl', features=256, out_channels=[256, 512, 1024, 1024], localhub=True)
depth_anything.load_state_dict(torch.load('checkpoints/depth_anything_vitl14.pth'))

transform = Compose([
    Resize(
        width=518,
        height=518,
        resize_target=False,
        keep_aspect_ratio=True,
        ensure_multiple_of=14,
        resize_method='lower_bound',
        image_interpolation_method=cv2.INTER_CUBIC,
    ),
    NormalizeImage(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
    PrepareForNet(),
])

image = cv2.cvtColor(cv2.imread('your image path'), cv2.COLOR_BGR2RGB) / 255.0
image = transform({'image': image})['image']
image = torch.from_numpy(image).unsqueeze(0)

# depth shape: 1xHxW
depth = depth_anything(image)
```

**Comment by LiheYoung** [comment]
Hi, this code snippet is for relative depth estimation. If you want to use Depth Anything for metric depth estimation, you need to follow the instructions in the [metric_depth](https://github.com/LiheYoung/Depth-Anything/tree/main/metric_depth) folder.

**Comment by MoAbbasid** [comment]
hi @LiheYoung, is there a way to use your pretrained metric depth models with training? I need to get metric values of pixel distance away from camera
